keyboards/handwired/footy/config.h

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define USB_VBUS_PIN GP9
+
```

Comment:
```suggestion
```

Reply:
I need to test this before removing it. It was pulled from the docs:

https://docs.keeb.io/stampy-usage-guide

Which say to set this

Reply:
`USB_VBUS_PIN` is only relevant for split keyboards.

Reply:
got it, removed